### PR TITLE
Some of the publications from Timo's list.

### PR DESCRIPTION
**Pull request by bangerth**
I deleted these from the google doc, and also removed a couple that didn't
use deal.II at all.

### CODE REVIEW

**Comment by tjhei** [comment]
thanks! /rebuild

**Comment by tjhei** [comment]
```
ERROR - BibTeX subsystem: /tmp/pP9_2zBisk/publications-2018.bib_312.utf8, line 1286, syntax error: found "Journal", expected end of entry ("}" or ")") (skipping to next "@")
```

**Comment by bangerth** [comment]
Works now.

File: publications-2019.bib [hunk 1]
```diff
@@ -319,6 +319,32 @@ @Article{ZC19
...
 
+@article{Ghesmati2018,
+  doi = {10.1515/jnma-2018-0047},
+  url = {https://doi.org/10.1515/jnma-2018-0047},
+  year = {2019},
+  month = nov,
+  publisher = {Walter de Gruyter {GmbH}},
+  volume = {accepted},
+  number = {0},
+  author = {A. Ghesmati and W. Bangerth and B. Turcksin},
```
Comment (tjhei) [inline publications-2019.bib]: wait, you are hiding one of your own papers in here? :-)

Comment (bangerth) [reply publications-2019.bib]: What did you think? ;-)

Comment (tjhei) [reply publications-2019.bib]: That you would immediately add your own papers once they are submitted/accepted. ;-)

Comment (bangerth) [reply publications-2019.bib]: It's actually a bit of an aggravating case: It was accepted in 2018 and has been stuck in "published online" for nearly a year now.

But you had it on your list :-)

**Pull request merged**